.circleci/scripts/deps-install.sh

```diff
@@ -5,7 +5,7 @@ set -x
 # Exit immediately if a command exits with a non-zero status.
 set -e
 
-yarn install --frozen-lockfile --har
+yarn install --frozen-lockfile #--har temporarily disabling for CI work
```

Comment:
Nit: If we're not tracking this work as something we'll be addressing, we can delete this comment
```suggestion
yarn install --frozen-lockfile
```